src/dotnet/Api/Contacts/Contact.cs

```diff
@@ -20,6 +20,7 @@ public sealed partial record Contact(
     [DataMember, MemoryPackOrder(3)] public Moment TouchedAt { get; init; }
     [DataMember, MemoryPackOrder(4)] public bool IsPinned { get; init; }
     [DataMember, MemoryPackOrder(7)] public string PeerContactName { get; init; } = "";
+    [DataMember, MemoryPackOrder(8)] public bool IsChatRoulette { get; init; }
```

Comment:
Doesn't look well, let's try to get rid of it

Reply:
Reworked with using SystemTag property